src/librustc_trans/builder.rs

```diff
@@ -1100,10 +1099,11 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
             llvm::LLVMRustBuildAtomicFence(self.llbuilder, order, scope);
         }
     }
-}
 
-fn check_call(typ: &str, llfn: ValueRef, args: &[ValueRef]) {
-    if cfg!(debug_assertions) {
+    fn check_call<'b>(&self,
```

Comment:
I think a comment here would be nice-- what is this `Cow` that is being returned?